scripts/cicd/verify-gather-logs-operations.py

```diff
@@ -39,13 +40,25 @@
     'starter-us-west-2'
 ]
 
+# This regex matches a valid hostname: an optional set of DNS labels each
```

Comment:
I like this simpler regex.  I would suggest adding to the comment that this matches against the expected format but does not ensure validity.  As such it's only input sanitization, not input validation.  Invalid IP addresses and non-existent hostnames will both be accepted.